src/sentry/static/sentry/app/components/progressRing.tsx

```diff
@@ -39,7 +37,7 @@ type Props = React.HTMLAttributes<SVGSVGElement> & {
    * The color of the ring bar. A function may be provided to compute the color
    * based on the percent value filled of the progress bar.
    */
-  progressColor?: string | ProgressColorFunction;
+  progressColor?: string;
```

Comment:
weren't we using it somewhere as a function?

Reply:
I searched for `progressColor` in both sentry and getsentry and wasn't able to find any function uses. Checking `git log -G progressColor` reveals that there used to be some, but I think they have been removed. We could change the API to allow proper aliases (similar to SvgIcon), but I'm not sure it makes sense to nail down an API when we don't actually have any users.

Reply:
yes. I think it's safe to remove it then :)